app/models/miq_worker/container_common.rb

```diff
@@ -105,12 +105,39 @@ def deployment_prefix
       "#{MiqServer.my_server.compressed_id}-"
     end
 
-    def worker_deployment_name
-      @worker_deployment_name ||= begin
-        deployment_name = abbreviated_class_name.dup.chomp("Worker").sub("Manager", "").sub(/^Miq/, "")
+    # move into abbreviated_class_worker_name?
+    def shortened_class_name(after)
+      after ? shortened_class_name_after : shortened_class_name_before
+    end
+
+    def shortened_class_name_before
+      abbreviated_class_name.dup.chomp("Worker")
+                            .sub("Manager", "")
+                            .sub(/^Miq/, "")
+    end
+
+    def shortened_class_name_after
+      # not thrilled about having an ibm specific abbreviation
+      # but that is the name they used in per https://www.ibm.com/products/power-virtual-server
+      abbreviated_class_name.dup.chomp("Worker")
+                            .gsub("Manager", "")
+                            .sub(/^Miq/, "")
+                            .gsub("Storage", "")
+                            .sub("EventHandler", "Events")
+                            .sub("EventCatcher", "Events")
```

Comment:
We're in `MiqWorker` here, could we override this and do `super.sub("EventCatcher", "Events")` so we don't have these all in core?